odo/backends/tests/test_csv.py

```diff
@@ -65,6 +65,13 @@ def test_pandas_read_supports_datetimes():
         assert df.dtypes['when'] == 'M8[ns]'
 
 
+def test_pandas_read_supports_whitespace_strings():
```

Comment:
What version of Pandas?  If I revert the fix and run the tests with Pandas `0.16.2`, it fails, as it should.

Pandas 0.17 fixes the `pd.to_datetime()` bug that we're working around here.

```
❯❯❯ py.test odo/backends/tests/test_csv_ws.py
========================================================= test session starts =========================================================
platform darwin -- Python 2.7.10, pytest-2.8.1, py-1.4.30, pluggy-0.3.1
rootdir: /Users/ksmith/work/odo, inifile:
collected 41 items

odo/backends/tests/test_csv.py ....F....................................

============================================================== FAILURES ===============================================================
____________________________________________ test_pandas_read_supports_whitespace_strings _____________________________________________

    def test_pandas_read_supports_whitespace_strings():
        with filetext('a,b, \n1,2, \n2,3, \n', extension='csv') as fn:
            csv = CSV(fn)
            ds = discover(csv)
>           assert ds == datashape.dshape("var * {a: int64, b: int64, '': ?string}")
E           assert dshape("var *...: ?datetime}") == dshape("var * ...'': ?string}")
E             Use -v to get the full diff

odo/backends/tests/test_csv.py:72: AssertionError
================================================= 1 failed, 40 passed in 0.36 seconds =================================================
```